.github/CODEOWNERS

```diff
@@ -15,6 +15,7 @@
 * @open-telemetry/collector-contrib-approvers
 
 cmd/mdatagen                                         @open-telemetry/collector-contrib-approvers @dmitryax
+cmd/telmetrygen                                      @open-telemetry/collector-contrib-approvers @mx-psi @amenasria @codeboten
```

Comment:
```suggestion
cmd/telemetrygen                                      @open-telemetry/collector-contrib-approvers @mx-psi @amenasria @codeboten
```

Reply:
Thanks ! Fixed in [cff8a3b](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/10427/commits/cff8a3b190f5bd1a190c055e1f800b3f5ef35e3e)